quicktask/app/Http/Controllers/TaskController.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Task;
+use Illuminate\Http\Request;
+
+class TaskController extends Controller
+{
+    public function show()
+    {
+        return view('tasks', [
+            'tasks' => Task::orderBy('created_at', 'asc')->get()
+        ]);
+    }
+
+    public function add(Request $request)
+    {
+        $validator = Validator::make($request->all(), [
+            'name' => 'required|max:255',
+        ]);
+
+        if ($validator->fails()) {
+            return redirect('/')
+                ->withInput()
+                ->withErrors($validator);
+        }
+
+        $task = new Task;
+        $task->name = $request->name;
+        $task->save();
+
+        return redirect('/');
+    }
+
+    public function delete($id)
+    {
+        Task::findOrFail($id)->delete();
```

Comment:
findOrFail() có tác dụng gì?

Reply:
Tìm đến đúng địa chỉ $id để xóa, nếu không tìm thấy sẽ trả ra kết quả not found